TCMSApp/src/client/app/blocks/fakers/fake-defects.factory.js

```diff
@@ -5,14 +5,16 @@
         .module('app.fakers')
         .factory('getDefects', getDefects);
 
+    getDefects.$inject = ['faker'];
 
-    function getDefects() {
```

Comment:
all these changes in the factory are to reach the usage as follows: `getDefects.fakeDefect(10)` instead of getDefects(10).
I like this variant too but it looks like other fakers act as: `getSomething(n)`
Have you had some special reason to use the `getDefects.fakeDefect(10)` variant?